docs/users/coaches.md

```diff
@@ -23,32 +23,33 @@ You can add new coaches using the _Add New Coach_ option, and edit them by click
 
 **Coach Status**: To distinguish coaches who are no longer working with the school, you can assign them the status of _exited_. Feedback and messages from these coaches are preserved, but they no longer have access to coach-specific features.
 
-## Assigning coaches to students
+## Assigning coaches to cohorts of a course
 
-Once a coach has been added to a school, they can immediately access all communities. However, to review submissions from students, they must be assigned to students in one of two ways:
+Once a coach has been added to a school, they can immediately access all communities. However, to review submissions from students, they must be assigned to a cohort of a course.
 
 ### Course Coaches
 
 To assign a coach to a course, head to the _Coaches_ sub-menu inside a course:
 
 ![Coaches assigned to a course](../assets/coaches/course_coaches_page.png)
 
-This assignment allows coaches to review submissions from _all_ students in a course. For courses with small numbers of students, this is probably sufficient configuration.
+This page lists all coaches who have been assigned to the course. To add a new coach, click on the _Assign Coaches to Course_ button. This will open a form that allows you to select coaches from the list of all coaches in the school and select the cohorts that they should be assigned to.
```

Comment:
`This page lists all coaches who have been assigned to the course. `

Should be something on the lines of 👇 

`This page lists all coaches who have been assigned to different cohorts in course.`